skxray/diff_roi_choice.py

```diff
@@ -289,97 +302,47 @@ def roi_rings_step(img_dim, calibrated_center, num_rings,
         else:
             raise ValueError("Provide step value for each q ring ")
 
-    # indices of rings
-    mesh = np.digitize(np.ravel(grid_values), np.array(ring_vals),
-                            right=False)
-
-    # to discard every-other bin and set the discarded bins indices to 0
-    mesh[mesh % 2 == 0] = 0
-    # change the indices of odd number of rings
-    indx = mesh > 0
-    mesh[indx] = (mesh[indx] + 1) // 2
-
-    (ring_inds, ring_vals, num_pixels,
-     pixel_list) = _process_rings(num_rings, img_dim,
-                                  ring_vals, mesh)
+    return ring_vals
 
-    return mesh.reshape(img_dim), ring_inds, ring_vals, num_pixels, pixel_list
 
-
-def _grid_values(img_dim, calibrated_center):
+def process_ring_edges(ring_vals):
     """
+    This function will provide edge values
+    of the each roi ring shape as (num_rings, 2)
+
     Parameters
     ----------
-    img_dim: tuple
-        shape of the image (detector X and Y direction)
-        Order is (num_rows, num_columns)
-        shape is [detector_size[0], detector_size[1]])
+    ring_vals : array
+        edge values of each ring
 
-    calibarted_center : tuple
-        defining the center of the image (column value, row value) (mm)
+    Returns
+    -------
+    ring_vals : array
+        edge values of each ring
+        shape is (num_rings, 2)
 
     """
-    xx, yy = np.mgrid[:img_dim[0], :img_dim[1]]
-    x_ = (xx - calibrated_center[1])
-    y_ = (yy - calibrated_center[0])
-    grid_values = np.float_(np.hypot(x_, y_))
+    ring_vals = np.asarray(ring_vals).reshape(-1, 2)
 
-    return grid_values
+    return ring_vals
 
 
-def _process_rings(num_rings, img_dim, ring_vals, ring_inds):
+def _grid_values(image_shape, calibrated_center):
```

Comment:
This is returning an array of radius, I thought we had this function is `skxray.core`?